ts/components/conversation/ContactName.tsx

```diff
@@ -10,20 +10,27 @@ interface Props {
   profileName?: string;
   i18n: LocalizerType;
   module?: string;
+  boldProfileName?: Boolean;
 }
 
 export class ContactName extends React.Component<Props> {
   public render() {
-    const { phoneNumber, name, profileName, i18n, module } = this.props;
+    const { phoneNumber, name, profileName, i18n, module, boldProfileName } = this.props;
     const prefix = module ? module : 'module-contact-name';
 
     const title = name ? name : phoneNumber;
     const shouldShowProfile = Boolean(profileName && !name);
-    const profileElement = shouldShowProfile ? (
-      <span className={`${prefix}__profile-name`}>
-        <Emojify text={profileName || ''} i18n={i18n} />
-      </span>
-    ) : null;
+    const profileElement = shouldShowProfile ?
+      (boldProfileName ? (
+        <span className={`${prefix}__profile-name`}>
+          <b><Emojify text={profileName || ''} i18n={i18n} /></b>
```

Comment:
I don't talk react that well, but maybe there's a way to do
```tsx
if (boldProfile) {
   profileElement = <b> {profileElement} </b>;
}
```
?
Maybe Mikunk can vouch?

Reply:
Yeah, that should work

Reply:
Chucked something else up but not sure how I feel about it